hack/libsubid_tag.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+if test $(${GO:-go} env GOOS) != "linux" ; then
+	exit 0
```

Comment:
This should also do 
 	echo no_libsubid

Reply:
amended